charts/spire/charts/tornjak-frontend/templates/deployment.yaml

```diff
@@ -44,10 +44,10 @@ spec:
               scheme: HTTP
               port: {{ .Values.service.port }}
             failureThreshold: 6
-            initialDelaySeconds: 120
-            periodSeconds: 45
+            initialDelaySeconds: 5
+            periodSeconds: 10
             successThreshold: 1
-            timeoutSeconds: 20
+            timeoutSeconds: 5
```

Comment:
I'd consider making these configurable. If its on a slower cluster, like a rpi, then it may need more time then other clusters.